targets/simple_switch_grpc/README.md

```diff
@@ -12,22 +12,35 @@ Github or the p4-dev mailing list.
 
  1. Follow instructions in the [PI
     README](https://github.com/p4lang/PI#dependencies) to install required
-    dependencies for the `--with-profo` configure flag - as well as for the
+    dependencies for the `--with-proto` configure flag - as well as for the
     `--with-sysrepo` configure flag if desired.
  2. Configure, build and install PI:
     ```
+    git clone https://github.com/p4lang/PI.git
+    git submodule update --init --recursive
+    cd PI
```

Comment:
Shouldn't the 'cd PI' command be _before_ the 'git submodule update ...' command?